lib/ui/shares/app_constants.dart

```diff
@@ -1,5 +1,8 @@
+import 'package:junkiri/models/task.dart';
+
 bool isFirstTime = true;
 String savedLocale = 'en';
 String currentUser = '';
+Task currentTask = Task(jobId: "", orderStatus: OrderStatus.CREATED, amount: 00, groceryList: ["Nothing"], achaarType: "Mango", pickedBy: "uid", paymentStatus: "paid");
```

Comment:
Where is this current task being used? Why are we hard coding these values?

Reply:
I had a issue on passing arguments on router. thats why I create a global variable for pass the value for next screen to check the flow of ui. It will be removed after fix that issue.

Reply:
fixed it